core/services/ocrcommon/transmitter_pipeline.go

```diff
@@ -86,7 +86,7 @@ func (t *pipelineTransmitter) CreateEthTransaction(ctx context.Context, toAddres
 	}
 
 	if run.State != pipeline.RunStatusCompleted {
-		return fmt.Errorf("unexpected pipeline run state: %s", run.State)
+		return fmt.Errorf("unexpected pipeline run state: %s with fatal errors %e", run.State, run.FatalErrors.ToError())
```

Comment:
%e is a scientific notation, you could use %w?

Reply:
Also will FatalErrors always be non-nil?

Reply:
FatalErrors is not a pointer, its type is a slice https://github.com/smartcontractkit/chainlink/blob/develop/core/services/pipeline/models.go#L45 

and toError is a function on non pointer https://github.com/smartcontractkit/chainlink/blob/develop/core/services/pipeline/models.go#L222-L234 so should be ok?